api/health/worker.go

```diff
@@ -146,9 +146,8 @@ func (w *worker) Results(tags ...string) (map[string]Result, bool) {
 	}
 
 	names := set.Set[string]{}
-	tagSet := set.NewSet[string](len(tags) + 1)
-	tagSet.Add(tags...)
-	tagSet.Add(ApplicationTag) // we always want to include the application tag
+	// we always want to include the application tag
+	tagSet := set.Of(append(tags, ApplicationTag)...)
```

Comment:
I'm always afraid of `append` keeping stuff around unexpectedly... I know that with `tags` here we know that `len(tags) == cap(tags)`... but it feels like unnecessary context for understanding the code... Also using `append` probably does an unnecessary copy unless the compiler is smart
```suggestion
	tagSet := set.Of(tags...)
	tagSet.Add(ApplicationTag) // we always want to include the application
```